mysql-test/suite/rocksdb/t/suite.opt

```diff
@@ -1 +1 @@
-$ROCKSDB_OPT $ROCKSDB_LOAD_ADD
+$ROCKSDB_OPT $ROCKSDB_LOAD_ADD --explicit-defaults-for-timestamp=1 --loose-rocksdb_lock_wait_timeout=1 --loose-rocksdb_strict_collation_check=0 --sql_mode=NO_ENGINE_SUBSTITUTION
```

Comment:
The default sql_mode in 5.7 is even stricter than that already, perhaps sql_mode setting here should be dropped